examples/zoned_io_async.c

```diff
@@ -473,10 +471,8 @@ static struct xnvmec_sub g_subs[] = {
 			{XNVMEC_OPT_QDEPTH, XNVMEC_LOPT},
 			{XNVMEC_OPT_DATA_INPUT, XNVMEC_LOPT},
 
-			{XNVMEC_OPT_DEV_NSID, XNVMEC_LOPT},
-			{XNVMEC_OPT_BE, XNVMEC_LOPT},
-			{XNVMEC_OPT_ADMIN, XNVMEC_LOPT},
 			{XNVMEC_OPT_ASYNC, XNVMEC_LOPT},
+			XNVMEC_BACKEND_OPTIONS ///< List of opts
```

Comment:
You can remove the backend options above ``XNVMEC_BACKEND_OPTIONS``, since ``XNVMEC_BACKEND_OPTIONS`` now adds them.